crates/bioauth-flow-rpc/src/errors/authenticate.rs

```diff
@@ -61,10 +63,35 @@ where
                 let (message, data) = map_txpool_error(err);
                 rpc_error_response::raw(api_error_code::TRANSACTION, message, data)
             }
+            Error::TxNotFinalized(err) => {
+                rpc_error_response::simple(api_error_code::TRANSACTION, err.to_string())
```

Comment:
Instead if a bare `err.to_string()`, consider using a `format!("transaction not finalized: {err}")`

Reply:
Should I apply this for rest errors here in this PR ?

Because we have `err.to_string()` for errors above.